smoke_tests/tests/authenticated/landingzone_definitions_tests.py

```diff
@@ -0,0 +1,39 @@
+import json
+
+from tests.smoke_test_case import SmokeTestCase
+
+"""Validates landingzone definitions endpoint"""
+
+
+class LandingZoneDefinitionsTests(SmokeTestCase):
+    CROMWELL_LZ_DEFINITION_NAME = "CromwellBaseResourcesFactory"
+
+    @staticmethod
+    def status_url() -> str:
+        return SmokeTestCase.build_lz_url("/api/landingzones/definitions/v1/azure")
+
+    """Validates that definitions endpoint return 401 when no token provided"""
+
+    def test_status_code_is_401_when_no_token_provided(self):
+        response = SmokeTestCase.call_lz(self.status_url())
+        self.assertEqual(response.status_code, 401)
+
+    """Validates lz definition response"""
+
+    def test_definitions(self):
```

Comment:
I wonder if these assertions are stricter than we need for a basic smoke test. The bare minimum would probably be a 200 status code and a non-empty list of definitions. Total length and the existence of the cromwell base landing zone doesn't seem necessary to me

Reply:
I wanted to validate our expectations regarding the type of LZs we are providing. It doesn't add any overhead to the test's execution time.

Reply:
At the same time if we are only checking for health, it might be not necessary.

Reply:
+1 for just checking 200 + non empty list.

Reply:
Will remove that part.